DashWallet/Sources/UI/CrowdNode/CrowdNode.storyboard

```diff
@@ -166,7 +167,7 @@
                                     <constraint firstAttribute="height" constant="46" id="9Hl-se-PhK"/>
                                 </constraints>
                                 <state key="normal" title="Button"/>
-                                <buttonConfiguration key="configuration" style="filled" image="image.crowdnode.notify" title="Close and notify when it’s done" imagePadding="24">
+                                <buttonConfiguration key="configuration" style="filled" image="bell.circle.fill" catalog="system" title="Close and notify when it’s done" imagePadding="24">
```

Comment:
Some icons are removed since we can use the system ones. Tried to do the same with the `bullet.point` icon and use `xmark` instead, but we need the "palette" render mode on it which doesn't seem to work on iOS 13.